src/mds/server/mds.cpp

```diff
@@ -118,6 +122,9 @@ void MDS::StartCompaginLeader() {
 }
 
 void MDS::Init() {
+    LOG_IF(FATAL, !CheckOrInsertBlockSize(etcdClient_.get()))
```

Comment:
Here Insert blocksize before  g_block_size is load from conf，Will cause a problem？